app/main/helpers/search_helpers.py

```diff
@@ -20,6 +22,33 @@ def get_page_from_request(request):
         return None
 
 
+def get_request_url_without_any_filters(request, filters):
+    '''
+        Logic:
+            -> get all url args
+            -> loop through the filters list and pop any matching url args
+            -> build and return url path
+
+        Arguements:
+            -> request - Request Object (import from flask)
+            -> filters - list of all filters from digital market place framework
+    '''
+
+    # stores all args from url path
+    all_request_filters = MultiDict(request.args.copy())
+
+    # loop to remove filters
+    for section in filters:
+        for _filter in filters[section]['filters']:
+            all_request_filters.poplist(_filter['name'])
+
+    # build url path with query string
```

Comment:
I don't think these three inline comments add a lot of value; they're describing what you're doing rather than why you're doing it. And we can see what you're doing. These comments could probably be removed.